morse_decode.rb

```diff
@@ -0,0 +1,62 @@
+@morse_dictionary = {
+  'a' => '.-',
+  'b' => '-...',
+  'c' => '-.-.',
+  'd' => '-..',
+  'e' => '.',
+  'f' => '..-.',
+  'g' => '--.',
+  'h' => '....',
+  'i' => '..',
+  'j' => '.---',
+  'k' => '-.-',
+  'l' => '.-..',
+  'm' => '--',
+  'n' => '-.',
+  'o' => '---',
+  'p' => '.--.',
+  'q' => '--.-',
+  'r' => '.-.',
+  's' => '...',
+  't' => '-',
+  'u' => '..-',
+  'v' => '...-',
+  'w' => '.--',
+  'x' => '-..-',
+  'y' => '-.--',
+  'z' => '--..',
+  ' ' => ' ',
+  '1' => '.----',
+  '2' => '..---',
+  '3' => '...--',
+  '4' => '....-',
+  '5' => '.....',
+  '6' => '-....',
+  '7' => '--...',
+  '8' => '---..',
+  '9' => '----.',
+  '0' => '-----'
+}
+
+def decode_char(char)
+  @current_morse_char = @morse_dictionary.invert[char]
+  @current_morse_char.upcase
+end
+
+def decode_word(word)
+  @entire_word = ''
+  @chars = word.split
+  @chars.each do |char|
+    @entire_word += decode_char(char)
+  end
+  print @entire_word.to_s
+end
```

Comment:
- [ ] You are almost there 👏 but currently you don't account for spaces in the words. When I try to decode a sentence, they are squished together. Kindly see below and try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/181471968-cbb222c0-5c2f-474a-a56f-c8a8dba55129.png)